native/ex_tokenizers/src/tokenizer.rs

```diff
@@ -27,19 +28,19 @@ impl ExTokenizersTokenizer {
     }
 }
 
-#[rustler::nif]
+#[rustler::nif(schedule = "DirtyCpu")]
 pub fn from_pretrained(identifier: &str) -> Result<ExTokenizersTokenizer, ExTokenizersError> {
     let tokenizer = Tokenizer::from_pretrained(identifier, None)?;
     Ok(ExTokenizersTokenizer::new(tokenizer))
 }
 
-#[rustler::nif]
+#[rustler::nif(schedule = "DirtyCpu")]
```

Comment:
Would this not be DirtyIo considering the potentially expensive file read ?

Reply:
Yep, 100%. My copy-paste got the best of me. 👍